drivers/scheduler/k8s/specs/postgres/px-postgres-storage.yaml

```diff
@@ -1,12 +1,45 @@
-##### Portworx storage class
+##### Portworx storage classes
 kind: StorageClass
 apiVersion: storage.k8s.io/v1
 metadata:
     name: px-postgres-sc
 provisioner: kubernetes.io/portworx-volume
 parameters:
+   fs: "ext4"
+   block_size: "512b"
```

Comment:
Also check the `ValidateCreateVolume` implementation in Portworx if all of these are getting verified there.